src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -139,30 +139,62 @@ public void readMessage(Entity entity, List<Message> messages, String user) {
     }
   }
   /** Stores the User in Datastore. */
- public void storeUser(User user) {
-  Entity userEntity = new Entity("User", user.getEmail());
-  userEntity.setProperty("email", user.getEmail());
-  userEntity.setProperty("aboutMe", user.getAboutMe());
-  datastore.put(userEntity);
- }
+  public void storeUser(User user) {
+    Entity userEntity = new Entity("User", user.getEmail());
+    userEntity.setProperty("email", user.getEmail());
```

Comment:
From line 142, there is an extra space at the start of the line.

Reply:
Yes - this is an example of unnecessary indentation. Please restore it to what it was previously. (The easiest way is probably to download the master branch from GitHub as a ZIP and copy over the existing lines.)